app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,12 @@
 
```

Comment:
![image](https://github.com/stephenkati/blog/assets/93039359/8a3626f8-0245-4d5c-b2ae-096ed1e64aa0)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalized 👍 to make your pull request professional 😊

Reply:
Great idea, thanks